src/poprox_recommender/evaluation/metrics/__init__.py

```diff
@@ -35,16 +37,21 @@ def measure_profile_recs(profile: ProfileRecs) -> list[dict[str, Any]]:
     Measure a single user profile's recommendations.  Returns the profile ID and
     a data frame of evaluation metrics.
     """
-    profile_id, all_recs, truth = profile
-    truth.index = truth.index.astype(str)
+    profile_id, all_recs, test = profile
+    test.index = test.index.astype(str)
+
+    filtered_test = test[test["rating"] > 0]
+    test_list = ItemList.from_df(filtered_test)
```

Comment:
If I remember how the test data is indexed correctly, I don't think this will work — I think it has the item IDs on the index, and they need to be in a column.

Reply:
May also need to change the test data loader.